site/content/en/v1.0.1/tasks/security/secure-gateways.md

```diff
@@ -2,16 +2,16 @@
 title: "Secure Gateways"
 ---
 
-This guide will help you get started using secure Gateways. The guide uses a self-signed CA, so it should be used for
-testing and demonstration purposes only.
+This task will help you get started using secure Gateways.
+This task uses a self-signed CA, so it should be used for testing and demonstration purposes only.
 
 ## Prerequisites
 
 - OpenSSL to generate TLS assets.
 
 ## Installation
 
-Follow the steps from the [Quickstart](../../quickstart) to install Envoy Gateway and the example manifest.
+Follow the steps from the [Quickstart](../quickstart) to install Envoy Gateway and the example manifest.
```

Comment:
ditto

Reply:
Actually, the new one is the more correct one.